cmake/modules/AddSwiftHostLibrary.cmake

```diff
@@ -17,10 +17,21 @@
 #
 # Remove once rdar://102202478 is fixed.
```

Comment:
Presumably we now need to use this function regardless of whether `rdar://102202478` is fixed?

Reply:
👍  Moved the comment to the `-dep.swift` hack.